e2e/deployment.go

```diff
@@ -240,6 +240,10 @@ type yamlResourceNamespaced struct {
 	// enable topology support (for RBD)
 	enableTopology bool
 	domainLabel    string
+
+	// enable read affinity support (for RBD)
+	enableReadAffinity  bool
+	crushLocationLabels string
```

Comment:
Hi @Madhu-1,
golangci-lint is faling because of maligned struct

```
e2e/deployment.go:233:29: struct of size 80 bytes could be of size 72 bytes:
struct{
	filename           	string,
	namespace          	string,
	domainLabel        	string,
	crushLocationLabels	string,
	oneReplica         	bool,
	enableTopology     	bool,
	enableReadAffinity 	bool,
}
``` (maligned)
type yamlResourceNamespaced struct {
```

Do we reorder as suggested?